pkgs/getzola/zola/registry.yaml

```diff
@@ -120,3 +120,5 @@ packages:
         overrides:
           - goos: windows
             format: zip
+        github_artifact_attestations:
+          signer_workflow: getzola/zola/.github/workflows/release.yml
```

Comment:
```suggestion
          signer-workflow: getzola/zola/.github/workflows/release.yml
```

This requires aqua v2.45.0-9 or later.

- https://github.com/aquaproj/aqua/pull/3593

To support older versions, we should use `signer-workflow` for now.
In future, we will replace `signer-workflow` with `signer_workflow`.